hbase-hadoop2-compat/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsReplicationSinkSourceImpl.java

```diff
@@ -49,6 +51,10 @@ public MetricsReplicationSinkSourceImpl(MetricsReplicationSourceImpl rms) {
     opsCounter.incr(batchsize);
   }
 
+  @Override public void incrFailedBatches(){
```

Comment:
nitpick: can you fix the whitespace here so override is on its own line? not sure what about our formatter is misconfigured, this happens for me too sometimes